app/facades/food_facade.rb

```diff
@@ -0,0 +1,12 @@
+class FoodFacade
+    def self.search_foods(ingredient)
+    json = FoodService.ingredient_search(ingredient)
+        json[:foods].map do |food|
+            Food.new(food)
+        end.first(10)
+    end
+
+    def self.total_results(ingredient)
+        FoodService.ingredient_search(ingredient)[:totalHits]
+    end
```

Comment:
To follow up on my last comment, in the search_foods method, you could return a 2d array like `[json[:totalHits, foods]` where `foods` is the collection of food objects you created on lines 4-6.